apps/src/templates/sectionProgressV2/ExpandedProgressDataColumn.jsx

```diff
@@ -15,17 +19,31 @@ function ExpandedProgressDataColumn({
   removeExpandedLesson,
 }) {
   const header = React.useMemo(() => {
+    // If there are only 2 levels, we only show the number so that the text fits the cell.
+    const headerText =
+      lesson.levels.length < 3
+        ? lesson.relative_position
+        : i18n.lesson() + ' ' + lesson.relative_position + ': ' + lesson.name;
```

Comment:
Concatenating translated strings will lead to strange results in cases where a language doesn't use the same word ordering/punctuation conventions as in English. Instead use a string with parameters. In fact we already have this one: https://github.com/code-dot-org/code-dot-org/blob/094fb729a635c051bb0601099e21a6f397662ca9/apps/i18n/common/en_us.json#L1413